pkg/utils/strings.go

```diff
@@ -27,34 +30,67 @@ const (
 
 var (
 	randomString     string
+	randomBytes      []byte
 	randomStringOnce sync.Once
 )
 
 func PreallocateRandomString(rnd Random, ln int) {
```

Comment:
shouldn't this function name change?

Reply:
Yes, but i didn't know how to call it better so, name stuck